scripts/install-deps.sh

```diff
@@ -49,7 +49,7 @@ main_linux()
 
 main_darwin()
 {
-    brew install freetype fontconfig qt5 harfbuzz boost pkg-config
+    brew install freetype fontconfig qt5 qt6 harfbuzz boost pkg-config
```

Comment:
Given the hope that this script is purely for CI and developers, I think including qt5 _and_ qt6 is fine.